app/mutations/FormChange/updateFormChange.ts

```diff
@@ -13,6 +13,11 @@ const mutation = graphql`
       formChange {
         id
         newFormData
+        operation
+        projectRevisionByProjectRevisionId {
+          ...ProjectContactForm_projectRevision
+          ...ProjectManagerFormGroup_revision
```

Comment:
This mutation is also used by the contact and operator forms...they don't need any of these fragments in the return type. Should we create a separate `updateFormChangeMutation` with different return queries, depending on the use case?